cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -76,6 +76,12 @@ spec:
            defaultMode: 420
            secretName: {{ .Values.kubecostModel.etlBucketConfigSecret }}
         {{- end }}
+        {{- if .Values.kubecostModel.federatedStorageConfigSecret }}
```

Comment:
For now yes, I think this is fine to avoid confusion between this "alpha" version of the feature and Thanos federation. Later on I think it can go in its own category, but I'll need to work though the design of that.